ocean_lib/web3_internal/contract_utils.py

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+import importlib
+import json
+import logging
+import os
+
+import artifacts  # noqa
```

Comment:
Strictly speaking, importing `artifacts` is not required.  I tried the unit tests without it and they all passed.

That said, I'm okay with importing it so that the import list presents an accurate picture of this file's dependencies, even those imported using `importlib`.

Reply:
that was basically my reasoning, too :)